tests/templates/System_Message.twig

```diff
@@ -0,0 +1 @@
+{$ message | trans() | raw $}
```

Comment:
This is a stubbed template used in the route tests. There is a [nicer version](https://backend.wikimedia.de/wiki/index.php?title=Web:Spendenseite-HK2013/rewrite/System_Message) in the wiki.